package.json

```diff
@@ -18,7 +18,15 @@
   },
   "sideEffects": false,
   "main": "dist/index.js",
-  "module": "dist/esm-browser/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
I'm wondering if this field now still makes sense? If I read https://webpack.js.org/configuration/resolve/#resolvemainfields correctly then, webpack will look for

- `['browser', 'module', 'main']` (in that order) when building for the browser.
- `['module', 'main']` when building for node.

Does that mean webpack will use the `module` entrypoint of the code when bundling for Node.js and transform everything into CommonJS for Node.js?

Do we agree that this has not yet anything to do with native ESM support for node?

Reply:
> Does that mean webpack will use the module entrypoint of the code when bundling for Node.js and transform everything into CommonJS for Node.js?

It does not transform into commonjs. ESM is used for more efficient bundling into one file. The result bundle can be any format configured in webpack (esm too since webpack v5)

> Do we agree that this has not yet anything to do with native ESM support for node?
No "module" field is bundler specific tool

Reply:
So yes, we do agree that this is something bundler specific then 😉 ?!

Reply:
So yes, we do agree that this is something bundler specific then 😉 ?!

Reply:
Yes, both module and browser fields are bundler specific.